NetKAN/Arkas-Config-Med-Clouds.netkan

```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
```

Comment:
`Arkas-Clouds-Medium` would be a more descriptive ID and help with location and sorting of the metadata files.